File: examples/AppModel/booksandrecords-spring-boot/src/main/java/com/ms/booksandrecords/service/dealevent/InvalidPrice.java
package com.ms.booksandrecords.service.dealevent;

import com.ms.booksandrecords.service.DealEvent;

public class InvalidPrice extends DealEvent {
    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public float getPrice() {
        return price;
    }

    public void setPrice(float price) {
        this.price = price;
    }

    private String id;
    private float price;

    public InvalidPrice(String id, float price) {
        this.id = id;
        this.price = price;
    }
}

File: examples/workspaces/booksandrecords-spring-boot/src/main/scala/com/ms/booksandrecords/service/dealevent/InvalidPrice.scala
package com.ms.booksandrecords.service.dealevent

import com.ms.booksandrecords.service.DealEvent

class InvalidPrice(var id: String, var price: Float) extends DealEvent {
  def getId = id

  def setId(id: String) = this.id = id

  def getPrice = price

  def setPrice(price: Float) = this.price = price
}